x-pack/legacy/plugins/apm/public/hooks/useKibanaUrl.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import url from 'url';
+import { useApmPluginContext } from './useApmPluginContext';
+
+export function useKibanaUrl(
+  /** The path to the plugin */ path: string,
+  /** The hash path */ hash: string
```

Comment:
The formatting looks odd. Shouldn't they either be after the argument:

```js
path: string // The path to the plugin
```

Or on the line above:
```js
// The path to the plugin
path: string
```

Either way I don't think they are  terribly helpful. So perhaps just remove them?

Reply:
It looks odd (to you) but it's valid JSDoc comments that show up when using autocomplete: 
![image](https://user-images.githubusercontent.com/9912/71007960-75711e00-20ad-11ea-82d1-a981d3d500be.png)